SpinCore_pp/config_parser_fn.py

```diff
@@ -231,6 +230,12 @@ class configuration(object):
             0,
             "number of field sweeps performed for a particular sample that day",
         ),
+        "misc_counter": (
+            int,
+            "file_names",
+            0,
+            "number of miscellaneous expts performed for a particular sample that day",
+        ),
```

Comment:
rather than making a counter specifically for calibrating pulses that will maybe be used once a year I make this a misc counter so that even with test equipment scripts we can use the active.ini